backend/uclapi/dashboard/webhook_views.py

```diff
@@ -0,0 +1,187 @@
+from django.http import JsonResponse
+from .models import App, User
+import keen
```

Comment:
Similar to the comment below, we never use keen directly now to avoid requests taking too long